ivy_tests/test_ivy/conftest.py

```diff
@@ -2,24 +2,57 @@
 import os
 import pytest
 from typing import Dict
+import sys
+import multiprocessing as mp
+
+
+mod_frontend = {
+    "tensorflow": None,
+    "numpy": None,
+    "jax": None,
+    "torch": None,
+    "mindspore": None,
+    "scipy": None,
+    "paddle": None,
+}  # multiversion
+mod_backend = {
+    "numpy": None,
+    "jax": None,
+    "tensorflow": None,
+    "torch": None,
+    "paddle": None,
+    "mxnet": None,
+}  # multiversion
+
```

Comment:
Why are those hard-coded values needed? can they be dynamic?

Reply:
These have an important role, these dictionaries are used to pass on the `multiprocess` objects across the testing code.